crates/bevy_gilrs/src/rumble.rs

```diff
@@ -0,0 +1,155 @@
+//! Handle user specified Rumble request events.
+use crate::converter::convert_gamepad_id;
+use bevy_app::EventReader;
+use bevy_core::Time;
+use bevy_ecs::{prelude::Res, system::NonSendMut};
+use bevy_input::gamepad::Gamepad;
+use bevy_log as log;
+use bevy_utils::HashMap;
+use gilrs::{ff, GamepadId, Gilrs};
+
+pub enum RumbleIntensity {
+    Strong,
+    Medium,
+    Weak,
+}
+impl RumbleIntensity {
+    fn effect_type(&self) -> ff::BaseEffectType {
+        use RumbleIntensity::*;
+        match self {
+            Strong => ff::BaseEffectType::Strong { magnitude: 63_000 },
+            Medium => ff::BaseEffectType::Strong { magnitude: 40_000 },
+            Weak => ff::BaseEffectType::Weak { magnitude: 40_000 },
```

Comment:
This magnitudes shouldn't be the same.

Reply:
In the [gilrs example](https://gitlab.com/gilrs-project/gilrs/-/blob/master/gilrs/examples/ff.rs) They use the same magnitude with a different `BaseEffectType` (you'll notice here we use `Strong` for `Medium` and `Weak` for `Weak`) to demonstrate a periodic change in force feedback intensity. Testing it myself with two different controllers, I find all different `RumbleIntensity` to be distinct enough.

Reply:
Didn't notice the different Types here. Then everything is fine here